prac_05/emails.py

```diff
@@ -0,0 +1,32 @@
+"""
+Emails
+Estimate: 30 minutes
+Actual: 50 minutes
+"""
+
+email = str(input("Email: "))
```

Comment:
Refactoring your document to segment into functions may result in easier to read code

Reply:
This applies many of your files - however I think this particular critique can be safely disregarded, as the length of most of these files is small. Therefore refactoring to add functions would make them longer